pkg/apis/certificates/validation/validation.go

```diff
@@ -304,7 +304,7 @@ func validateCertificateSigningRequestUpdate(newCSR, oldCSR *certificates.Certif
 			case len(newConditions) > len(oldConditions):
 				validationErrorList = append(validationErrorList, field.Forbidden(field.NewPath("status", "conditions"), fmt.Sprintf("updates may not add a condition of type %q", t)))
 			case !apiequality.Semantic.DeepEqual(oldConditions, newConditions):
-				conditionDiff := cmp.Diff(oldConditions, newConditions)
+				conditionDiff := reflect.DeepEqual(oldConditions, newConditions)
```

Comment:
Same comment as https://github.com/kubernetes/kubernetes/pull/129161/files#r1880534624

Maybe encapsulate this in a function with its own test, to make sure we handle lists properly (especially nil vs empty slice vs populated)?